rawdisk/plugins/filesystems/linux.yapsy-plugin

```diff
@@ -0,0 +1,8 @@
+[Core]
+Name = LinuxPlugin
```

Comment:
Actually, `linux` is the partition type 0x83. So I am not sure if it should be renamed.